lib/modules/landing/landing_presenter.dart

```diff
@@ -5,6 +5,21 @@ abstract class LandingPresenter
 
 class LandingPresenterImpl extends LandingPresenter
     implements LandingInteractorDelegate, LandingViewDelegate {
+  LandingPresenterImpl() {
+    stateDidInit.voidListen(_stateDidInit);
+    alertDialogDidClose.voidListen(_alertDialogDidClose);
+    authenticationDidFailToValidate.listen(_authenticationDidFailToValidate);
+
+    final List<Stream<dynamic>> streams = [
+      animationDidFinish,
+      authenticationDidValidate
+    ];
+
+    streams.zip().listen((event) {
```

Comment:
I think you can do something like `animationDidFinish.zipWith(authenticationDidValidate).map{ it[1] }.listen()` right? 😛  I play a lot with rx in android so usually I prefer chaining with all the possible operators rather than creating interim variables.

Reply:
The real implement in dart 👌 
```dart
animationDidFinish
    .zipWith<bool,bool>(authenticationDidValidate, (t, s) => s)
      .listen();
```